README.md

```diff
@@ -1,2 +1,5 @@
 # git-demo
 to demonstrate how to use git and GitHub
+
+
+Swaha Bhattacharya - https://github.com/Swaha294
```

Comment:
Could you please add a `-` so this becomes a list item?

Reply:
Also, please make your name into a link that goes to that URL. The URL should not be visible.